translator.cjs

```diff
@@ -364,7 +364,9 @@ class Translator {
       'en',
       'ko',
       'pt',
+      'pt-BR',
```

Comment:
Since `pt-BR` and `ar-SA` are locale that did not exist originally, there are no directories for them and I am getting an error.


```
> babyagi-ui@0.1.0 translate
> node ./translator.cjs

[ATi18n]:> Would you like me to translate the entire project automatically to all the locales you have enabled? (y/n) n
[ATi18n]:> Would you like to synchronize translations? (y/n) y
[ATi18n]:> Enter the source language code: en
node:fs:1451
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, scandir '/Users/yoshiki/Works/next/babyagi-ui/public/locales/pt-BR'
    at Object.readdirSync (node:fs:1451:3)
    at Translator.getTranslationsFromDir (/Users/yoshiki/Works/next/babyagi-ui/translator.cjs:623:22)
    at Translator.synchronizeTranslations (/Users/yoshiki/Works/next/babyagi-ui/translator.cjs:577:39)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /Users/yoshiki/Works/next/babyagi-ui/translator.cjs:763:17 {
  errno: -2,
  syscall: 'scandir',
  code: 'ENOENT',
  path: '/Users/yoshiki/Works/next/babyagi-ui/public/locales/pt-BR'
}

Node.js v18.12.1
```

Reply:
This will be corrected on the project side after this PR is merged.

Reply:
Sorry about it, I was working on a solution and just copy pasted the modified version of the locales array. It will be automatically detected from or the .env file or from the i18n configuration instance's locales array.